beets/plugins.py

```diff
@@ -518,7 +518,7 @@ def feat_tokens(for_artist=True):
     feat_words = ["ft", "featuring", "feat", "feat.", "ft."]
     if for_artist:
         feat_words += ["with", "vs", "and", "con", "&"]
-    return r"(?<=\s)(?:{})(?=\s)".format(
+    return r"(?<=[\s,(,\[])(?:{})(?=\s)".format(
```

Comment:
Not sure if my regex knowledge is enough here but it looks like with `[\s,(,\[]` you try to say "match either whitespace or bracket open or square bracket open". Why are there comma (`,`) characters in between? Are they really required?  Really not sure and did not look it up but from the top of my head this should suffice: `[\s(\[]`

Reply:
Obviously yes. Thanks for spotting this. I have no idea why I put a comma in there.